test/support/api/flows.js

```diff
@@ -77,16 +80,14 @@ Cypress.Commands.add('routeFlow', (mutator = _.identity) => {
     .as('routeFlow');
 });
 
-Cypress.Commands.add('routePatientFlows', (mutator = _.identity, patientId = '1') => {
```

Comment:
This default to id `1` was very problematic.  Created many patient models that were included with the same id.  It made some tests flaky when this or the action version resolved last.